### DISCUSSION TITLE
Updating in bulk selection doesn't work

### DISCUSSION THREAD

**Discussion by The1Waterman** [firefly-iii/firefly-iii#9415] [General]
Updating in bulk selection doesn't work, either for category's or tags, (tick box unchecked for category's and 'add these tags' selected

**Comment by The1Waterman** [comment]
OK, tags have to be created first, probably the same for category's?

**Comment by JC5** [comment]
Hey, thanks for starting a discussion. 

I can't replicate what you are running into. New and existing tags and categories can be added to transactions without any problems. 

You can try this on the demo site and see for yourself.
